packages/line-ending-selector/lib/main.js

```diff
@@ -1,12 +1,16 @@
 'use babel'
 
 import _ from 'underscore-plus'
-import {CompositeDisposable, Disposable} from 'atom'
+import { CompositeDisposable, Disposable } from 'atom'
 import SelectListView from 'atom-select-list'
 import StatusBarItem from './status-bar-item'
 import helpers from './helpers'
 
 const LineEndingRegExp = /\r\n|\n/g
+
+// the following regular expression is executed natively via the `substring` package,
+// where `\A` corresponds to the beginning of the string.
+// eslint-disable-next-line no-useless-escape
```

Comment:
@jasonrudolph, @smashwilson can you verify if what I wrote here is correct? 🙃

Reply:
@rafeca: That sounds mostly right. I see that the `\A` was introduced in https://github.com/atom/line-ending-selector/pull/56. The explanation in that PR might help you craft the comment above and/or you could consider including a link to that PR. Does that help?

Reply:
Yes, thank you!